docs/extend/extend-apiml/api-mediation-oidc-authentication.md

```diff
@@ -0,0 +1,87 @@
+# Zowe API Mediation Layer OIDC authentication
+
+Zowe API ML can be configured to authenticate users by accepting Access Tokens issued by an external Identity Provider (IDP) implementing OIDC/OAuth2 protocols.
+This is useful in advanced deployments of Zowe, where client applications need to access mainframe as well as enterprise/distributed systems, while offering their users the comfort of SSO across the systems boundaries.  
+
+This article provides details of the API ML OIDC authentication functionality and related configuration needed to achieve it.
+
+- [OIDC Usage](#oidc-usage)
+- [Authentication Workflow](#authentication-flow)
+- [Configuration](#api-ml-oidc-configuration)
+    * [Prerequisites](#prerequisites)
+    * [OIDC provider configuration](#oidc-provider-configuration)
+    * [API ML GW configuration](#api-ml-gw-configuration)
+    * [SAF/ESM configuration](#esm-configuration)
+- [Troubleshooting](#troubleshooting)
+
+## OIDC Usage
+OIDC authentication is used together with the z/OS [Identity Propagation](https://www.redbooks.ibm.com/redbooks/pdfs/sg247850.pdf) feature to implement the AI ML Identity Federation.
+[OIDC](https://openid.net/specs/openid-connect-core-1_0.html) protocol adds an authentication layer on top of the [OAuth2](https://www.rfc-editor.org/rfc/rfc6749) Authorization protocol.
+It is used by APIML client applications to verify user's identity with a distributed OIDC provider trusted by the mainframe security.
+After successful login the OIDC provider grants the client an Access Token (along with Identity Token, eventually also a Refresh Token if requested and appropriate).
+The Access Token issued by a trusted OIDC provider is now accepted by APIML.
+is the foundation of API ML multi-domain Identity Federation.
```

Comment:
"is the foundation of API ML multi-domain Identity Federation." is an incomplete sentence. Please check.

Reply:
Removed the sentence.